### PR TITLE
Add youtube icon and link

### LINKED ISSUE #397: Add the youtube logo to the contact me section
**Is your feature request related to a problem? Please describe.**
Currently, the youtube logo and my channel is not there in the contact me section

**Describe the solution you'd like**
Add the youtube logo and the link my channel to that logo
Channel link :- https://www.youtube.com/channel/UCiuDD_uA4v5PF2cr-3nTChA

**Describe alternatives you've considered**
Don't forget to subscribe my channel 🔥

**Are you work on this?**
No

### PR DESCRIPTION
**Pull request by ad-astra-per-ardua**
## Purpose

The purpose of this Pull Request is to fix #397 

## Goals

Added youtube icon with link.
To enhance code readability, line breaks have been added

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by ad-astra-per-ardua** [comment]
Sorry for the delay.

File: README.md [hunk 1]
```diff
@@ -269,7 +269,16 @@ const info = {
...
+<a href="https://github.com/YoshithaRathnayake"><img src="https://img.icons8.com/fluency/48/000000/github.png" alt="GitHub Icon" title="GitHub"/></a> 
+<a href="https://gitlab.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/gitlab.png" alt="GitLab Icon" title="GitLab"/></a> 
+<a href="https://twitter.com/Yoshitha2008"><img src="https://img.icons8.com/color/48/000000/twitter-circled--v1.png" alt="Twitter Icon" title="Twitter"/></a> 
+<a href="https://www.instagram.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/instagram-new.png" alt="Instagram Icon" title="Instagram"/></a> 
+<a href="https://www.linkedin.com/in/yoshitha-rathnayake"><img src="https://img.icons8.com/fluency/48/linkedin.png" alt="LinkedIn Icon" title="LinkedIn"/></a> 
+<a href="https://yoshitharathnayake.medium.com"><img src="https://img.icons8.com/ios-filled/48/ffffff/medium--v2.png" alt="Medium Icon" title="Medium"/></a>
+<a href="https://www.youtube.com/channel/UCiuDD_uA4v5PF2cr-3nTChA"><img width="48" height="48" src="https://img.icons8.com/color/48/youtube-play.png" alt="Youtube Icon"/></a>
```
Comment (YoshithaRathnayake) [inline README.md:280]: There is already given the size for the icon in the link so no need to add them manually right? @ad-astra-per-ardua 
What happened before you add the size? If there is no change by removing the width and the height remove them.

```suggestion
<a href="https://www.youtube.com/channel/UCiuDD_uA4v5PF2cr-3nTChA"><img src="https://img.icons8.com/color/48/youtube-play.png" alt="Youtube Icon"/></a>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (YoshithaRathnayake) [inline README.md:280]: And for the youtube link, add the below link

https://www.youtube.com/@YoshithaRathnayake
```suggestion
<a href="https://www.youtube.com/@YoshithaRathnayake"><img width="48" height="48" src="https://img.icons8.com/color/48/youtube-play.png" alt="Youtube Icon"/></a>
```

Do not forget to subscribe my channel and follow my github account 😃

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -269,7 +269,16 @@ const info = {
...
+<a href="https://github.com/YoshithaRathnayake"><img src="https://img.icons8.com/fluency/48/000000/github.png" alt="GitHub Icon" title="GitHub"/></a> 
+<a href="https://gitlab.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/gitlab.png" alt="GitLab Icon" title="GitLab"/></a> 
+<a href="https://twitter.com/Yoshitha2008"><img src="https://img.icons8.com/color/48/000000/twitter-circled--v1.png" alt="Twitter Icon" title="Twitter"/></a> 
+<a href="https://www.instagram.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/instagram-new.png" alt="Instagram Icon" title="Instagram"/></a> 
+<a href="https://www.linkedin.com/in/yoshitha-rathnayake"><img src="https://img.icons8.com/fluency/48/linkedin.png" alt="LinkedIn Icon" title="LinkedIn"/></a> 
+<a href="https://yoshitharathnayake.medium.com"><img src="https://img.icons8.com/ios-filled/48/ffffff/medium--v2.png" alt="Medium Icon" title="Medium"/></a>
+<a href="https://www.youtube.com/@YoshithaRathnayake"><img src="https://img.icons8.com/color/48/youtube-play.png" alt="Youtube Icon"/></a>
+
```
Comment (YoshithaRathnayake) [inline README.md:281]: Remove this extra line @ad-astra-per-ardua 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -269,7 +269,15 @@ const info = {
...
+<a href="https://github.com/YoshithaRathnayake"><img src="https://img.icons8.com/fluency/48/000000/github.png" alt="GitHub Icon" title="GitHub"/></a> 
+<a href="https://gitlab.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/gitlab.png" alt="GitLab Icon" title="GitLab"/></a> 
+<a href="https://twitter.com/Yoshitha2008"><img src="https://img.icons8.com/color/48/000000/twitter-circled--v1.png" alt="Twitter Icon" title="Twitter"/></a> 
+<a href="https://www.instagram.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/instagram-new.png" alt="Instagram Icon" title="Instagram"/></a> 
+<a href="https://www.linkedin.com/in/yoshitha-rathnayake"><img src="https://img.icons8.com/fluency/48/linkedin.png" alt="LinkedIn Icon" title="LinkedIn"/></a> 
+<a href="https://yoshitharathnayake.medium.com"><img src="https://img.icons8.com/ios-filled/48/ffffff/medium--v2.png" alt="Medium Icon" title="Medium"/></a>
+<a href="https://www.youtube.com/@YoshithaRathnayake"><img src="https://img.icons8.com/color/48/youtube-play.png" alt="Youtube Icon"/></a>
```
Comment (YoshithaRathnayake) [inline README.md:280]: Add the title, do not remove it @ad-astra-per-ardua 
```suggestion
<a href="https://www.youtube.com/@YoshithaRathnayake"><img src="https://img.icons8.com/color/48/youtube-play.png" alt="YouTube Icon" title="YouTube"/></a>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (YoshithaRathnayake) [inline README.md:280]:
```suggestion
<a href="https://www.youtube.com/@YoshithaRathnayake"><img src="https://img.icons8.com/color/48/youtube-play.png" alt="YouTube Icon"/></a>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — APPROVED** [review]
Good job!
Thank you @ad-astra-per-ardua 
Don't forget to subscribe my youtube channel and follow me on github and social media

**Pull request merged**